frontend/src/components/pages/acls/Acl.List.tsx

```diff
@@ -486,11 +485,10 @@ const RolesTab = observer(() => {
           variant="outline"
           onClick={() => appGlobal.history.push('/security/roles/create')}
           {...getCreateUserButtonProps()}
-          isDisabled={api.userData?.canCreateRoles === false || !api.isAdminApiConfigured}
+          isDisabled={api.userData?.canCreateRoles === false}
```

Comment:
We should check for the endpoint compatibility hook here. It should be there in the form of `Features.RBAC`.

Endpoint equals to `/redpanda.api.console.v1alpha1.SecurityService`, Method: `POST` in case it doesn't exist yet.

Reply:
Updated